library/lib.sh

```diff
@@ -0,0 +1,266 @@
+#!/bin/bash
+# vim: dict=/usr/share/beakerlib/dictionary.vim cpt=.,w,b,u,t,i,k
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+#
+#   Description: Library for system roles tests
+#   Author: Sergei Petrosian <spetrosi@redhat.com>
+#
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+#   library-prefix = library
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+rolesInstallAnsible() {
+    if rlIsRHELLike 7; then
+        # Hardcode to the only supported version on EL 7
+        ANSIBLE_VER=2.9
+    fi
+    if rlIsFedora || (rlIsRHELLike ">7" && [ "$ANSIBLE_VER" != "2.9" ]); then
+        rlRun "dnf install python$PYTHON_VERSION-pip -y"
+        rlRun "python$PYTHON_VERSION -m pip install ansible-core==$ANSIBLE_VER.*"
+    elif rlIsRHELLike 8; then
+        PYTHON_VERSION=3.9
+        rlRun "dnf install python$PYTHON_VERSION -y"
+        rlRun "python$PYTHON_VERSION -m pip install ansible==$ANSIBLE_VER.*"
+    else
+        # el7
+        rlRun "yum install ansible-$ANSIBLE_VER.* -y"
+    fi
+}
+
+rolesCloneRepo() {
+    local role_path=$1
+    if [ ! -d "$REPO_NAME" ]; then
```

Comment:
```suggestion
    if [ ! -d "$role_path" ]; then
```
I think `REPO_NAME` is just the name of the role repo, not a full path.  Unless somewhere the code guarantees that the pwd is the parent directory of `role_path`, this may not work as intended.